bin/git-feature

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-branch_prefix=feature
+branch_prefix=$(git config --default feature --get git-extras.feature.prefix)
```

Comment:
Look like `--default` only exists in newer version of `git`. I gets `error: unknown option default` when running this command. My git version is `git version 2.17.1`.

Reply:
Okay, thanks for testing for me, will rework this.